Main.java

```diff
@@ -1,6 +1,7 @@
+import java.util.Scanner;
 /**
  *

```

Comment:
Don't forget the description in the code header. Otherwise, this is excellent work!